app/main/views/invites.py

```diff
@@ -105,7 +105,7 @@ def accept_org_invite(token):
     if existing_user:
         invited_org_user.accept_invite()
         if existing_user not in organisation_users:
-            invited_org_user.add_to_organisation()
+            invited_org_user.add_to_organisation(existing_user.id)
```

Comment:
You can change this without having to change the model
```suggestion
            existing_user.add_to_organisation()
```